src/api.rs

```diff
@@ -399,6 +400,12 @@ pub struct EncryptedTempKey {
     _internal_fp12: Fp12Elem<Fp256>,
 }
 
+impl Hashable for EncryptedTempKey {
+    fn to_bytes(&self) -> ByteVector {
+        self._internal_fp12.to_bytes()
```

Comment:
Would it make sense to just use `.bytes()` here?
```suggestion
        self.bytes().to_vec()
```